backend/src/Service/Order/OrderService.php

```diff
@@ -35,6 +37,8 @@ public function createOrder(OrderCreateRequest $request): OrderResponse|CanCreat
         if($order) {
 
          $this->subscriptionService->updateRemainingOrders($request->getStoreOwner());
+
+         $this->notificationLocalService->createNotificationLocal($request->getStoreOwner()->getStoreOwnerId(), NotificationConstant::NEW_ORDER_TITLE, NotificationConstant::CREATE_ORDER_SUCCESS, $order->getId());
```

Comment:
We can define a separate function just for handling the creation of a local notification, and injecting notification create request in the create function of notification service.